open_dread_rando/patcher_editor.py

```diff
@@ -85,5 +85,19 @@ def remove_entity(self, reference: dict, map_category: typing.Optional[str]):
         if map_category is not None:
             self.get_scenario_map(reference["scenario"]).raw.Root[map_category].pop(actor_name)
     
+    def copy_actor_groups(self, scenario_name: str, base_actor_name: str, new_actor_name: str):
+        """
+        Copies a base actor's groups to a new actor's groups
```

Comment:
worth noting in the description that this needs to be done within the same scenario

Reply:
bump

Reply:
fixed